python/ray/data/datasource/file_meta_provider.py

```diff
@@ -196,11 +196,6 @@ def expand_paths(
         paths: List[str],
         filesystem: "pyarrow.fs.FileSystem",
     ) -> Tuple[List[str], List[Optional[int]]]:
-        logger.warning(
```

Comment:
Ah, I think I commented this before that it should probably not log unconditionally.